recipes/openldap/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "openldap" %}
+{% set version = "2.4.46" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: ftp://ftp.openldap.org/pub/OpenLDAP/openldap-release/openldap-{{ version }}.tgz
+  sha256: 9a90dcb86b99ae790ccab93b7585a31fbcbeec8c94bf0f7ab0ca0a87ea0c4b2d
+
+build:
+  number: 0
+  skip: True  # [not unix]
+  run_exports:
+    - {{ pin_subpackage('openldap', max_pin='x.x') }}
+  script:
+    # disable server components
+    - ./configure --prefix=$PREFIX  --disable-slapd --disable-slurpd
+    - make
+    - make install 
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+  host:
+    - openssl
+    - cyrus-sasl
+  run:
+    - openssl
+    - cyrus-sasl  
+
+test:
+  commands:
+    - test $PREFIX/include/ldap.h   # [unix]
+  # imports:
+  #   - ldap
+  #   - ldap.controls
+  #   - ldap.extop
+  #   - ldap.schema
+  #   - slapdtest
+
+about:
+  home: https://www.openldap.org
+  license: The OpenLDAP Public License
```

Comment:
Looks like the canonical name for this version would be `OLDAP-2.8`:

https://spdx.org/licenses/OLDAP-2.8.html

IANAL, but it smells an awful lot like 

https://spdx.org/licenses/BSD-3-Clause.html

So the argument could be made for a `license_family` of BSD, which is backed up by [prior art in anaconda](https://anaconda.org/anaconda/openldap/files/modal/info/58b6178beb3ae60a96660bcc), but OTHER is probably just fine...